src/tests/test-data/user-sessions.json

```diff
@@ -1,8 +1,14 @@
 [
   {
-    "id" : 1,
-    "user_id" : 4,
-    "refresh_token" : "IqrTkSKmel",
-    "refresh_token_expires_at" : "2025-11-21 19:19:40.911+03"
+    "id": 2,
```

Comment:
Can't start with id: 1 because of [this](https://github.com/codex-team/notes.api/blob/main/src/repository/storage/postgres/orm/sequelize/userSession.ts#L68).